src/components/Asset.vue

```diff
@@ -240,9 +240,17 @@ export default {
         text.push(this.from);
       }
       return text.join(' ');
+    },
+    openAssetsUids () {
+      return this.$store.state.stateQueryParameters.assets;
     }
   },
   created() {
+    if (this.openAssetsUids.indexOf(this.uid) > -1) {
```

Comment:
I assume you don't need a separate computet prop here if it's just used once?

```suggestion
    if (this.$store.state.stateQueryParameters.assets.indexOf(this.uid) > -1) {
```